flexmeasures/auth/tests/test_perm.py

```diff
@@ -49,3 +55,38 @@ def test_consultant_user_update_perm(
         has_access = False
 
     assert has_access == has_perm
+
+
+@pytest.mark.parametrize(
+    "requesting_user, account_name, required_perm, has_perm",
+    [
+        # Consultant tries to update client account
+        ("test_consultant@seita.nl", "Test ConsultancyClient Account", "update", True),
+        # Consultant tries to update account from another client
+        ("test_consultant@seita.nl", "Test Supplier Account", "update", False),
+    ],
+)
+def test_consultant_account_update_perm(
```

Comment:
If this test case is only about the "update" permission, then the permission ("update"') does not need to be part of the parameters.

Or you make this test case more general, allowing to (later) test also other permissions of consultants on accounts. I think that might be the nicer way.